templates/starter/terraform/repo.yaml

```diff
@@ -0,0 +1,37 @@
+templateApi: 1.0.0
+metadata:
+  type: repo
+  name: azd-terraform-starter
+  description: Starter template with Bicep as infrastructure provider for Azure Developer CLI (azd).
+
+repo:
+  includeProjectAssets: true
+
+  remotes:
+    - name: azure-samples-main
+      url: git@github.com:Azure-Samples/azd-terraform-starter.git
```

Comment:
Please change starters to `azd-starter-{thing}` that way we can always have starter at the beginning and expand to more tech in the future.

Reply:
Yeah - let's do that.